assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -158,55 +158,66 @@ function GoogleAnalyticsSettings({
           {siteKitDisplayText}
         </SettingSubheading>
       </div>
-      <div>
-        <InlineForm>
-          <VisuallyHiddenLabel htmlFor="gaTrackingId">
-            {TEXT.ARIA_LABEL}
-          </VisuallyHiddenLabel>
-          <SettingsTextInput
-            aria-label={TEXT.ARIA_LABEL}
-            id="gaTrackingId"
-            value={analyticsId}
-            onChange={handleUpdateId}
-            onKeyDown={handleOnKeyDown}
-            placeholder={TEXT.PLACEHOLDER}
-            hasError={Boolean(inputError)}
-            hint={inputError}
-            disabled={analyticsActive}
-          />
-          <SaveButton
-            type={BUTTON_TYPES.SECONDARY}
-            size={BUTTON_SIZES.SMALL}
-            disabled={disableSaveButton}
-            onClick={handleOnSave}
+      {installed && analyticsActive ? (
```

Comment:
```suggestion
      {analyticsActive ? (
```

that should work. If not, need to check whether the plugin is _active_, not just _installed_